src/controller/config.js

```diff
@@ -39,7 +40,7 @@ dropArea.addEventListener('dragover', (event) => {
 dropArea.addEventListener('drop', (event) => {
     event.preventDefault();
     event.stopPropagation();
-    if(!isBodyPartSaved){
+    if(!isBodyPartSaved && Object.keys(coordinatesMap).length > 0){
```

Comment:
If added check for coordinates map. If there are no coordinates then there is no body part. So it would not be saved